src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -56,6 +56,80 @@
                                        ["code", "lib", "func", "reason"])
 
 
+def _encode_asn1_int(backend, x):
+    i = backend._lib.ASN1_INTEGER_new()
+    # i = backend._ffi.gc(i, backend._lib.ASN1_INTEGER_free)
```

Comment:
This should be removed and a comment should be added at the top of this function stating that the generated object will not be garbage collected (to help remind us that we need to either know it's being added to something that will own it or that we need to manually register for gc).